src/schema/unique-constraint-builder.ts

```diff
@@ -22,6 +18,46 @@ export class UniqueConstraintNodeBuilder implements OperationNodeSource {
       UniqueConstraintNode.cloneWith(this.#node, { nullsNotDistinct: true }),
     )
   }
+
+  deferrable() {
+    return new UniqueConstraintNodeBuilder(
+      UniqueConstraintNode.cloneWith(this.#node, { deferrable: true }),
+    )
+  }
+
+  notDeferrable() {
+    return new UniqueConstraintNodeBuilder(
+      UniqueConstraintNode.cloneWith(this.#node, { deferrable: false }),
+    )
+  }
+
+  initiallyDeferred() {
+    return new UniqueConstraintNodeBuilder(
+      UniqueConstraintNode.cloneWith(this.#node, {
+        initiallyDeferred: true,
+      }),
+    )
+  }
+
+  initiallyImmediate() {
+    return new UniqueConstraintNodeBuilder(
+      UniqueConstraintNode.cloneWith(this.#node, {
+        initiallyDeferred: false,
```

Comment:
nitpicky

Alternatively, we could go for `initially: 'deferred' | 'immediate'` to be more explicit.

Reply:
Two options --> I'd go with a boolean. Also property called `initially` is not very specific. You need to look at the type to know what it's related to.